core/bones/base.py

```diff
@@ -493,13 +528,47 @@ def unserialize(self, skel: 'viur.core.skeleton.SkeletonInstance', name: str) ->
         """
         if name in skel.dbEntity:
             loadVal = skel.dbEntity[name]
+            if self.compute:  # handle this in the first place
+                if self.compute.threshold.method == ThresholdMethods.Until:  # we maybe must compute the value new
+                    # if we have no value we set it to now and compute new
+                    valid_until = skel.dbEntity.get(f"{name}__vaild_until", utils.utcNow())
```

Comment:
I would rather store the last updated date than the until date. If you change the bone parametrization and reduce the interval (maybe you accidentally provides minutes, instead of seconds) this affects only the next computation.